hi I'm ccyygg on steam

cyg6695 opened the pull request:
[Korean.zip](https://github.com/AUTOMATIC1111/Radiology/files/2921995/Korean.zip)
this is Korean translation to Rimworld mod Radiology
Is making a pull request right like this?

Commit:
Added Ear and Eye mutations

Commit:
Fixed incorrect endings

Commit:
Minor Development

Commit:
Organ mutation development

Commit:
Tweaks

Commit:
Rename

AUTOMATIC1111 committed:
Merge pull request #2 from Galax-Git/master

Mutation development

AUTOMATIC1111 committed:
Merge branch 'development'

AUTOMATIC1111 committed:
readme and a fix

AUTOMATIC1111 committed:
cancer and irradiate job improvement

Reveal incorrectly diagnosed on death
Kill/destroy only at 100%
Bad diagnosis mention
Redo diagnosis operation
Irradiate job cancelling

AUTOMATIC1111 committed:
wording fix

AUTOMATIC1111 committed:
better handling for blink outside bounds

AUTOMATIC1111 committed:
mutation disabling multiple skills

AUTOMATIC1111 committed:
mutations that make the pawn glow

AUTOMATIC1111 committed:
added carapace and psionic shield mutations without graphics

AUTOMATIC1111 committed:
Full Carapace and Psionic shield Mutations

AUTOMATIC1111 committed:
HediffMutationDef refnamed and horns mutation added

AUTOMATIC1111 committed:
Night Dweller mutation

AUTOMATIC1111 committed:
second-hand irradiation and special room type

AUTOMATIC1111 committed:
irradiating prisoners

AUTOMATIC1111 committed:
mutations: fire shield and regenration

AUTOMATIC1111 committed:
info icon for mutations in pawn's health card

AUTOMATIC1111 committed:
further development

Additional blockers
Battery mutation
Bloodlust mutation

AUTOMATIC1111 committed:
descriptions, thoughts added; irradiation sound changed to persist

AUTOMATIC1111 committed:
minor additions to setskill mutation

AUTOMATIC1111 committed:
new mutations, new shield graphic

AUTOMATIC1111 committed:
whatever just release it

AUTOMATIC1111 committed:
fix add skill bug; fix glower performance bug

AUTOMATIC1111 committed:
fixed horns generating for wild men and robots

AUTOMATIC1111 committed:
-

AUTOMATIC1111 committed:
chinese translation

AUTOMATIC1111 committed:
prevent traders from selling horns

AUTOMATIC1111 commented:
Well... no, that's not it. This is the same as just sharing the archive, without using the github at all.

Here's how a pull request should look:
https://github.com/AUTOMATIC1111/Rimlaser/pull/1 (this is a person adding translation for another one of my mods for the first time)
https://github.com/AUTOMATIC1111/Rimlaser/pull/2 (same person updating his translation)

His changes are seen as commits on the pull request page, and after I accept the pull request, he is correctly labeled as author of those files in my repository.
In your repository, there isn't even a Korean directory in the translations: https://github.com/cyg6695/Radiology/tree/master/Languages

I'll offer this again: if you can't/don't want to fight with github, I can accept the zip file and add the translation of the game to repository myself.

AUTOMATIC1111 committed:
possible fix for can't build bug

AUTOMATIC1111 committed:
updated harmony

AUTOMATIC1111 committed:
1.1 support
removing/adding mutated parts to colonists
Glow: fixed bug deleting the colonist when he enters the map
Volatile: reworked: no more chain explosions
Regeneration: now heals permanent injuries

AUTOMATIC1111 committed:
some more 1.1 stuff

AUTOMATIC1111 committed:
new mutations
radiation extender
mutation monitor
work type changed to basic
fixed choppy irradiate sound
many small mutation fixes

AUTOMATIC1111 committed:
1.2

AUTOMATIC1111 committed:
1.3
